session/bootstrap.go

```diff
@@ -428,6 +428,11 @@ const (
 	CreateMDLView = `CREATE OR REPLACE VIEW mysql.tidb_mdl_view as (
 	select JOB_ID, DB_NAME, TABLE_NAME, QUERY, SESSION_ID, TxnStart, TIDB_DECODE_SQL_DIGESTS(ALL_SQL_DIGESTS, 4096) AS SQL_DIGESTS from information_schema.ddl_jobs, information_schema.CLUSTER_TIDB_TRX, information_schema.CLUSTER_PROCESSLIST where ddl_jobs.STATE = 'running' and find_in_set(ddl_jobs.table_id, CLUSTER_TIDB_TRX.RELATED_TABLE_IDS) and CLUSTER_TIDB_TRX.SESSION_ID=CLUSTER_PROCESSLIST.ID
 	);`
+	// CreateStatsTableLocked stores the locked tables
+	CreateStatsTableLocked = `CREATE TABLE IF NOT EXISTS mysql.stats_table_locked(
+		table_id bigint(64) NOT NULL,
+		PRIMARY KEY (table_id)
+	);`
```

Comment:
Do we need a command to see which tables are locked? `mysql.stats_table_locked` shows `table_id` rather than table name, which is not very convenient. cc @winoros

Reply:
I add table_schema and table_name to mysql.stats_table_locked.

Reply:
Do not put able_name here, since it could be changed by ALTER command. It's better to have a "show" command and join table_id to get table_name behind the command. 
@xuyifangreeneyes we can think about making the show command show lock status and analyze options together.

Reply:
Aleady add show command, please review.